debezium-connector-sqlserver/src/test/java/io/debezium/connector/sqlserver/SqlServerConnectorTest.java

```diff
@@ -43,6 +45,78 @@ public void testValidateUnableToConnectNoThrow() {
         }
     }
 
+    @Test
+    public void shouldReturnSingleTaskConfig() {
+        Map<String, String> config = new HashMap<>();
+        config.put(SqlServerConnectorConfig.HOSTNAME.name(), "narnia");
+        config.put(SqlServerConnectorConfig.PORT.name(), "4321");
+        config.put(SqlServerConnectorConfig.DATABASE_NAME.name(), "sqlserver1");
+        config.put(SqlServerConnectorConfig.DATABASE_NAMES.name(), "sqlserver1,sqlserver2");
+        config.put(SqlServerConnectorConfig.USER.name(), "pikachu");
+        config.put(SqlServerConnectorConfig.PASSWORD.name(), "raichu");
+
+        Config validated = connector.validate(config);
+        connector.start(config);
+        List<Map<String, String>> taskConfigs = connector.taskConfigs(1);
+        assertThat(taskConfigs.size()).isEqualTo(1);
+        assertThat(taskConfigs.get(0).get(SqlServerConnectorConfig.DATABASE_NAME.name())).isEqualTo("sqlserver1");
+        assertThat(taskConfigs.get(0).get(SqlServerConnectorConfig.DATABASE_NAMES.name())).isEqualTo("sqlserver1,sqlserver2");
+    }
+
+    @Test
+    public void shouldReturnTwoTaskConfigs() {
+        Map<String, String> config = new HashMap<>();
+        config.put(SqlServerConnectorConfig.HOSTNAME.name(), "narnia");
+        config.put(SqlServerConnectorConfig.PORT.name(), "4321");
+        config.put(SqlServerConnectorConfig.DATABASE_NAME.name(), "sqlserver1");
+        config.put(SqlServerConnectorConfig.DATABASE_NAMES.name(), "sqlserver1,sqlserver2");
+        config.put(SqlServerConnectorConfig.USER.name(), "pikachu");
+        config.put(SqlServerConnectorConfig.PASSWORD.name(), "raichu");
+
+        Config validated = connector.validate(config);
+        connector.start(config);
+        List<Map<String, String>> taskConfigs = connector.taskConfigs(2);
+        assertThat(taskConfigs.size()).isEqualTo(2);
+        assertThat(taskConfigs.get(0).get(SqlServerConnectorConfig.DATABASE_NAME.name())).isEqualTo("sqlserver1");
+        assertThat(taskConfigs.get(0).get(SqlServerConnectorConfig.DATABASE_NAMES.name())).isEqualTo("sqlserver1");
+        assertThat(taskConfigs.get(1).get(SqlServerConnectorConfig.DATABASE_NAME.name())).isEqualTo("sqlserver2");
+        assertThat(taskConfigs.get(1).get(SqlServerConnectorConfig.DATABASE_NAMES.name())).isEqualTo("sqlserver2");
+    }
+
+    @Test
+    public void shouldReturnTaskConfigWithoutDatabaseNames() {
```

Comment:
Do we really need two ways of communicating the list of databases to a task? I'd expect a list of one database passed to the task. The singular `DATABASE_NAME` in the context of a task is a temporary (previous) behavior.

Reply:
I removed this test as it'll be deprecated once the case for removing the dependency on database.dbname is implemented.